driver-core/src/main/com/mongodb/internal/operation/FindOperation.java

```diff
@@ -847,6 +874,9 @@ private BsonDocument getCommand(final SessionContext sessionContext) {
         if (showRecordId) {
             commandDocument.put("showRecordId", BsonBoolean.TRUE);
         }
+        if (allowDiskUse && (description == null || serverIsAtLeastVersionThreeDotTwo(description))) {
```

Comment:
Does this mean that `explain` will always include `allowDiskUse` regardless of server version?  Is that what we want?  Maybe it doesn't matter since we only support explain in the legacy API, and legacy API doesn't support allowDiskUse.  What do you think?

Reply:
Updated it's now only included if allowDiskUse is set to true and its server version > 3.2